taichi/ir/type.cpp

```diff
@@ -87,6 +87,14 @@ std::string TensorType::to_string() const {
   return s;
 }
 
+int TensorType::vector_width() const {
+  int vw = 1;
+  for (auto dim : shape_) {
+    vw *= dim;
+  }
+  return vw;
+}
+
 int Type::vector_width() const {
   return 1;  // TODO: CPU vectorization
```

Comment:
@strongoier correct me if I were wrong, but I think we can probably remove this "TODO" since we have a dedicated TensorType now?

Reply:
We should actually remove all legacy code related to CPU vectorization.